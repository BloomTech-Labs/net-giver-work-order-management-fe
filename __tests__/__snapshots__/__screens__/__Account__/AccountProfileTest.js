import React from 'react'
import renderer from 'react-test-renderer'
import AccountSettings from '../../../../screens/Account/AccountSetting'

test('renders correctly', () => {
    const tree = renderer.create(<AccountSettings />).toJSON()
    expect(tree).toMatchSnapshot()
})
