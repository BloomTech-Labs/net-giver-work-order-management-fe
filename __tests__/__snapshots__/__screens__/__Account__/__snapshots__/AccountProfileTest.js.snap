// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    AccountSetting
  </Text>
</View>
`;
